lnrpc/websocket_proxy.go

```diff
@@ -239,12 +272,17 @@ func (p *WebsocketProxy) upgradeToWebSocketProxy(w http.ResponseWriter,
 
 		// Whenever a pong message comes in, we extend the deadline
 		// until the next read is expected by the interval plus pong
-		// wait time.
+		// wait time. Since we can never _reach_ any of the deadlines,
```

Comment:
> we can never _reach_ any of the deadlines

So the deadlines were never reached before?

Reply:
We did reach the deadlines before, because we failed to extend them. And reaching a deadline means we get the `i/o read` or `i/o write` error. So we always need to push away the deadline to avoid failing.